packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -2829,18 +2836,30 @@ class _WidgetInspectorState extends State<WidgetInspector>
         onPanUpdate: _handlePanUpdate,
         behavior: HitTestBehavior.opaque,
         excludeFromSemantics: true,
-        child: IgnorePointer(
-          ignoring: isSelectMode,
-          key: _ignorePointerKey,
-          child: widget.child,
+        child: ValueListenableBuilder<bool>(
```

Comment:
Repeating the value listenable builder with the same key is a bit of a code smell. Some options: move the ValueListenableBuilder to the root of the build method (the couple extra widgets that rebuild won't be a perf issue) or achieve the same effect by listening for changes to the valueListenable by registering a listener in initState and removing it in dispose.
You should also check whether this whole stateful widget can now be made stateless.

Reply:
unless there is a way to remove the lastPointerLocation field, this is probably not feasible so I would instead switch to adding listeners in initState and disposing them in dispose.